plugins/module_utils/network/iosxr/config/interfaces/interfaces.py

```diff
@@ -193,6 +195,7 @@ def _state_overridden(self, want, have):
 
         for each in have:
             for interface in want:
+                interface["name"] = normalize_interface(interface["name"])
```

Comment:
This is probably redundant. The `have` interfaces _should_ always be optimized since these are read from the device.

Reply:
but if you check the condition is for want not for have.its for 2nd for loop which is iterating on want

Reply:
Oh, my bad.